tests/browser_test.go

```diff
@@ -99,8 +99,19 @@ func TestTmpDirCleanup(t *testing.T) {
 
 	b.Close()
 
-	matches, err = filepath.Glob(tmpDirPath + "/xk6-browser-data-*")
-	assert.NoError(t, err)
+	// We need to wait for something (k6 browser, chromium or the os) to
+	// actually complete the removal of the directory. It's a race condition.
+	// To try to mitigate the issue, we're adding a retry which waits half a
+	// second if the dir still exits.
```

Comment:
Nice comment. I'd also add `TODO` note here, saying, fix this race condition - or might even make sense to create an issue for it.